contracts/deploy/055_curve_amo.js

```diff
@@ -0,0 +1,478 @@
+const {
+  deploymentWithGuardianGovernor,
+  impersonateAccount,
+  sleep,
+} = require("../utils/deploy");
+const addresses = require("../utils/addresses");
+const hre = require("hardhat");
+const { BigNumber, utils, Contract } = require("ethers");
+const {
+  getAssetAddresses,
+  getOracleAddresses,
+  isMainnet,
+  isFork,
+  isMainnetOrFork,
+} = require("../test/helpers.js");
+const { MAX_UINT256 } = require("../utils/constants");
+
+// 5/8 multisig
+const guardianAddr = addresses.mainnet.Guardian;
+
+module.exports = deploymentWithGuardianGovernor(
+  { deployName: "055_curve_amo" },
+  async ({ deployWithConfirmation, ethers, getTxOpts, withConfirmation }) => {
+    const { deployerAddr, governorAddr } = await getNamedAccounts();
+    const sDeployer = await ethers.provider.getSigner(deployerAddr);
+
+    let {
+      actions,
+      tokenAddress,
+      poolAddress,
+      crvRewards,
+      gaugeAddress,
+      poolId,
+    } = await deployCurve({
+      deployWithConfirmation,
+      withConfirmation,
+      ethers,
+    });
+
+    const { cHarvester, actions: harvesterActions } =
+      await deployHarvesterAndOracleRouter({
+        deployWithConfirmation,
+        withConfirmation,
+        ethers,
+      });
+
+
+    actions = actions.concat(harvesterActions);
+
+    // actions = actions.concat(await reDeployOETH({
+    //   deployWithConfirmation,
+    //   withConfirmation,
+    //   ethers
+    // }));
+
+    actions = actions.concat(
+      await configureDripper({
+        deployWithConfirmation,
+        withConfirmation,
+        ethers,
+        cHarvester
+      })
+    );
+
+    actions = actions.concat(
+      await deployConvexETHMetaStrategy({
+        deployWithConfirmation,
+        withConfirmation,
+        ethers,
+        tokenAddress,
+        poolAddress,
+        gaugeAddress,
+        poolId,
+        crvRewards,
+        cHarvester,
+      })
+    );
+
+    // Governance Actions
+    // ----------------
+    return {
+      name: "Deploy Curve AMO, Harvester, Dripper and Oracle Router",
+      actions,
+    };
+  }
+);
+
+const configureDripper = async ({
+  deployWithConfirmation,
+  withConfirmation,
+  ethers,
+  cHarvester
+}) => {
+  const { deployerAddr } = await getNamedAccounts();
+  const sDeployer = await ethers.provider.getSigner(deployerAddr);
+
+  const cVaultProxy = await ethers.getContract("OETHVaultProxy");
+  const cDripperProxy = await ethers.getContract("OETHDripperProxy");
```

Comment:
I'm not sure how this works if the dripper isn't deployed already?

Reply:
I has already been deployed as part of [this deploy](https://github.com/OriginProtocol/origin-dollar/pull/1334) but not configured.